src/genie/libs/parser/iosxe/c9500/show_platform.py

```diff
@@ -55,12 +55,14 @@ class ShowVersionSchema(MetaParser):
                     'model_rev_num': str,
                     'mb_rev_num': str,
                     'model_num': str,
-                    'system_sn': str,
+                    Optional ('system_sn'): {
```

Comment:
sorry, why change is needed? Please share sample output of `show version`

Reply:
thank you for the output. I confirmed no SN in output.
```
System Serial Number               :
```
but this doesn't require to change the schema. let's change to `Optional` without having new level.
```
Optional('system_sn'): str,

Reply:
confirmed